cmake/externals/dlib.cmake

```diff
@@ -3,10 +3,7 @@ FetchContent_Declare(dlib
     # there is non an official tag which supports STFT,
     # choose a relatively stable commit id for that.
     GIT_TAG        a12824d42584e292ecb3bad05c4b32c2015a7b89
+    SOURCE_SUBDIR  not_set
```

Comment:
”SOURCE_SUBDIR not_set“ 
Is it needed?

Reply:
Yes, it's intentional for the library which we want to use it in a C++ header-only way.